Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -257,8 +266,46 @@ extension BrowserViewController: TopToolbarDelegate {
     return url
   }
   
+  @discardableResult
+  func handleIPFSSchemeURL(_ url: URL, visitType: VisitType) -> Bool {
+    guard !PrivateBrowsingManager.shared.isPrivateBrowsing else {
+      topToolbar.leaveOverlayMode()
+      let errorPageHelper = ErrorPageHelper(certStore: nil)
```

Comment:
Why is this initialized with `nil`?
Can you get the one associated with the tab already or initialize this with the proper store?

Reply:
same as above